Cargo.toml

```diff
@@ -11,13 +11,16 @@ repository = "https://github.com/JonahPlusPlus/bevy_atmosphere"
 exclude = ["/assets/", "/examples/", "/.github/"]
 
 [dependencies]
-bevy = {version = "0.10", default-features = false, features = ["bevy_asset", "bevy_render", "bevy_pbr"]}
-bevy_atmosphere_macros = {path = "macros", version = "0.2"}
+bevy = {version = "0.11.0", default-features = false, features = ["bevy_asset", "bevy_render", "bevy_pbr"]}
+bevy_atmosphere_macros = { path = "macros" }
+bevy_spectator = "0.2.0"
```

Comment:
I think this crate should be only in dev-dependencies.

Reply:
yes you're right